### PR TITLE
database schema

### LINKED ISSUE #34: Create Schema.sql script
## Task Description

- Create Schema file that generates the tables in the RDS

## User Stories

- As an engineer, we need to create a schema.sql file so that we can generate the tables into the rds.

## Relevant Files [If Available] 

-

### PR DESCRIPTION
**Pull request by ayeshaa63**
Resolves #34

### CODE REVIEW

File: database/schema.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
+CREATE TABLE "judges"(
+    "judge_id" SMALLINT NOT NULL,
+    "name" TEXT NOT NULL,
+    "appointed" DATE NOT NULL,
+    "circuit_id" SMALLINT NULL,
+    "judge_type_id" BIGINT NOT NULL,
+    "gender" VARCHAR(1) NOT NULL
```
Comment (RachelPun) [inline database/schema.sql:7]: i think gender and circuit_id at leat should allow NULL, maybe appointed too?

Comment (ayeshaa63) [reply database/schema.sql:7]: Yes agreed, I'll change it now.

File: database/schema.sql [hunk 2]
```diff
@@ -0,0 +1,45 @@
+CREATE TABLE "judges"(
+    "judge_id" SMALLINT NOT NULL,
+    "name" TEXT NOT NULL,
+    "appointed" DATE,
+    "circuit_id" SMALLINT,
+    "judge_type_id" BIGINT NOT NULL,
+    "gender" VARCHAR(1)
```
Comment (RachelPun) [inline database/schema.sql:7]: CHAR(1)

File: database/schema.sql [hunk 3]
```diff
@@ -0,0 +1,45 @@
+CREATE TABLE "judges"(
+    "judge_id" SMALLINT NOT NULL,
+    "name" TEXT NOT NULL,
```
Comment (RachelPun) [inline database/schema.sql:3]: maybe VARCHAR(50)?

File: database/schema.sql [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    "appointed" DATE,
+    "circuit_id" SMALLINT,
```
Comment (RachelPun) [inline database/schema.sql:5]: REFERENCES circuits (circuit_id)?

File: database/schema.sql [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    "judge_type_id" BIGINT NOT NULL,
```
Comment (RachelPun) [inline database/schema.sql:6]: SMALLINT?

**Review by RachelPun — COMMENTED** [review]
reviewed

**Review by ErvinRex — APPROVED** [review]
Nice work, good updates made after the comments. This schema fits our erd perfectly.

**Pull request merged**